package.json

```diff
@@ -1,21 +1,25 @@
 {
   "name": "ambrosus-node-contracts",
```

Comment:
I haven't had an opportunity to pose this question before, but should it really be named `"ambrosus-node-contracts"`? I got that the whole repo is named like this, but I can't really see what it has to do with the `node`? Can't it rather be just `"ambrosus-contracts"`? 😛

Reply:
On the other hand this are contracts related to operating an atlas/hermes/apollo node aren't they?